lib/trento_web/views/prometheus_view.ex

```diff
@@ -10,7 +10,7 @@ defmodule TrentoWeb.PrometheusView do
 
   def render("target.json", %{target: target}) do
     %{
-      targets: ["#{target.ssh_address}:#{@node_exporter_port}"],
+      targets: ["#{List.first(target.ip_addresses, target.hostname)}:#{@node_exporter_port}"],
```

Comment:
Use the 1st obtained IP address (or the hostname as fallbak) as node exporter address. I think it should be enough, but if in the future we need more customization, we will need to add back some settings to be added by the user maybe

Reply:
Could we track this in a new card as well?

Reply:
Done